inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/reader/RedisReader.java

```diff
@@ -0,0 +1,445 @@
+                        return null;
+                    }
+                })
+                .registerTypeAdapter(KeyStringValueSet.class, new TypeAdapter<KeyStringValueSet>() {
+                    @Override
+                    public void write(JsonWriter out, KeyStringValueSet kv) throws IOException {
+                        out.beginObject();
+                        out.name("key").value(new String(kv.getKey()));
+                        out.name("value").beginArray();
+                        for (byte[] bytes : kv.getValue()) {
+                            out.value(new String(bytes));
+                        }
+                        out.endArray();
+                        out.endObject();
+                    }
+
+                    @Override
+                    public KeyStringValueSet read(JsonReader in) throws IOException {
+                        return null;
+                    }
+                })
+                .registerTypeAdapter(KeyStringValueString.class, new TypeAdapter<KeyStringValueString>() {
+                    @Override
+                    public void write(JsonWriter out, KeyStringValueString kv) throws IOException {
+                        out.beginObject();
+                        out.name("key").value(new String(kv.getKey()));
+                        out.name("value").value(new String(kv.getValue()));
+                        out.endObject();
+                    }
+
+                    @Override
+                    public KeyStringValueString read(JsonReader in) throws IOException {
+                        return null;
+                    }
+                })
+                .registerTypeAdapter(KeyStringValueZSet.class, new TypeAdapter<KeyStringValueZSet>() {
+                    @Override
+                    public void write(JsonWriter out, KeyStringValueZSet kv) throws IOException {
+                        out.beginObject();
+                        out.name("key").value(new String(kv.getKey()));
+                        out.name("value").beginArray();
+                        for (ZSetEntry entry : kv.getValue()) {
+                            out.beginObject();
+                            out.name("element").value(new String(entry.getElement()));
+                            out.name("score").value(entry.getScore());
+                            out.endObject();
+                        }
+                        out.endArray();
+                        out.endObject();
+                    }
+
+                    @Override
+                    public KeyStringValueZSet read(JsonReader in) throws IOException {
+                        return null;
+                    }
+                })
+                .create();
+    }
+
+    /**
+     * init replicator's commandParser
+     */
+    private void initReplicator() {
+        redisReplicator.addCommandParser(CommandName.name("APPEND"), new DefaultCommandParser());
```

Comment:
can we reuse  DefaultCommandParser()  here ?

Reply:
Each command has its own commandParser, so I have to change it into unified parser

Reply:
I my test we can reuse DefaultCommandParser().